src/public/manifest.json

```diff
@@ -11,6 +11,22 @@
         "name": "Debug",
         "scope": "embeddable",
         "guiAuthoring": false
+      },
+      {
+        "label": "debugQuestionWrapper",
+        "name": "Debug Question Wrapper",
+        "scope": "embeddable-decoration",
+        "decorates": [
+          "Embeddable::MultipleChoice",
+          "Embeddable::OpenResponse",
+          "Embeddable::ImageQuestion",
+          "ManagedInteractive",
+          "MwInteractive",
+          "ImageInteractive",
+          "VideoInteractive"
+        ],
+        "guiAuthoring": true,
```

Comment:
If this now provides a gui for authoring, can we set the guiAuthoring for the embeddable scope to true as well?

Reply:
That was copy/pasted from teacher-edition manifest, so both should presumably be set to false since I don't think there's any support for GUI authoring of the debug plugin.